mssql_column.py

```diff
@@ -0,0 +1,24 @@
+
+def validate_column(target_column, target_table, cursor):
+    """ Makes sure column belongs to table
+
+    Args:
+         target_column (str): column in question
+         target_table (str): table name
+         cursor (pyodbc.Cursor): active cursor
+
+    Returns:
+        validated column name
+
+    Raises:
+         ValueError if target_column is not valid in target_table
+    """
+    valid_columns = []
+    for column in cursor.columns(target_table):
```

Comment:
This can probably be future work but I think we can probably cache this result. If I'm understanding it correctly, this will execute a query on the table to fetch the columns every time we want to execute a query on it. The table structure is unlikely to change that often, seems like we could do this on block start and cache the results.